lib/v2/blocs/deeplink/viewmodels/deeplink_event.dart

```diff
@@ -31,4 +31,13 @@ class ClearDeepLink extends DeeplinkEvent {
 
   @override
   String toString() => 'ClearDeepLink';
-}
\ No newline at end of file
+}
+
+class HandleIncomingSigningRequest extends DeeplinkEvent {
+  final String link;
+
+  const HandleIncomingSigningRequest(this.link);
+
+  @override
+  String toString() => 'HandleIncomingSigningRequest';
```

Comment:
should we add the link as part of the toString?

Reply:
cool yeah... although I guess it's for debugging

Reply:
Done